ui/components/app/edit-gas-fee-popover/edit-gas-fee-popover.test.js

```diff
@@ -186,4 +188,26 @@ describe('EditGasFeePopover', () => {
     });
     expect(screen.queryByText('10% increase')).toBeInTheDocument();
   });
+
```

Comment:
Test cases look redundant, it will be nicer to test that advance defaults are actually visible on edit gas fee modal.